test/test_block_token.py

```diff
@@ -119,13 +119,47 @@ def test_code_fence_with_backticks_and_tildes_in_the_info_string(self):
         self._test_match(block_token.CodeFence, lines, arg, language='aa')
 
 
+class TestCodeFenceBehavior(unittest.TestCase):
```

Comment:
Tests seem OK, yet I would simplify the nesting: below, `test_content_property()` is nested within `TestHTMLBlock`, I would nest all analogous `test_content_property()` like this.

Reply:
I moved all the property getter unit tests to a dedicated class. Is that how you meant?

Reply:
Not quite like that, but how you did it, I think it is OK. Of course, there is always room for some de-duplication, but I think it is good enough for now. :)

Reply:
Ok! So, with that I think all comments should be resolved.